scenario/objectsearch.go

```diff
@@ -20,12 +20,12 @@ type (
 
 	// ObjectSearchSettings ObjectSearch search listbox, field or master dimension
 	ObjectSearchSettings struct {
-		ID           string             `json:"id" doc-key:"objectsearch.id"`
-		SearchTerms  []string           `json:"searchterms" doc-key:"objectsearch.searchterms"`
-		SearchType   ObjectSearchType   `json:"type" doc-key:"objectsearch.type"`
-		SearchSource ObjectSearchSource `json:"source" doc-key:"objectsearch.source"`
-		ErrorOnEmpty bool               `json:"erroronempty" doc-key:"objectsearch.erroronempty"`
-		Filename     helpers.RowFile    `json:"searchtermsfile" doc-key:"objectsearch.searchtermsfile"`
+		ID           string             `json:"id" doc-key:"objectsearch.id" displayname:"ID"`
+		SearchTerms  []string           `json:"searchterms" doc-key:"objectsearch.searchterms" displayname:"Search terms"`
+		SearchType   ObjectSearchType   `json:"type" doc-key:"objectsearch.type" displayname:"Search type"`
+		SearchSource ObjectSearchSource `json:"source" doc-key:"objectsearch.source" displayname:"Search source"`
+		ErrorOnEmpty bool               `json:"erroronempty" doc-key:"objectsearch.erroronempty" displayname:"Throw error if no search results"`
+		Filename     helpers.RowFile    `json:"searchtermsfile" doc-key:"objectsearch.searchtermsfile" displayname:"Filename"`
```

Comment:
Add displayelement:"file" to the tags and it will populate the GUI with a file browser.